docs/pipeline-components-and-applications/iglu/igluctl-2/index.md

```diff
@@ -268,6 +269,31 @@ Example:
 }
 ```
 
+## static verify-parquet
+
+`igluctl static verify-parquet` verifies that schema is evolved correctly within the same major version (e.g. from `1-a-b` to `1-c-d`) for parquet transformation (for loading into Databricks). It reports the breaking schema versions.
+
+It accepts one required arguments:
+
+- `input` - path to your schema files.
+
+```bash
+$ ./igluctl static verify-parquet /path/to/static/registry
+```
```

Comment:
Could you add example output?

Reply:
Just checking — you would still add output right? (Not just the command example.)

Reply:
Yep, added it in the new commit.